thealgorithms/data_structures/stacks/__init__.py

```diff
@@ -1,23 +0,0 @@
-class Stack:
```

Comment:
Hmm, not sure about outright removing this stack. Maybe we should add it in a `simple_stack.py` module, or something.

Reply:
This part of code is not being used any where

Reply:
Most code is not being used anywhere, it's a library.

Reply:
Added the file as `simple_stack.py`